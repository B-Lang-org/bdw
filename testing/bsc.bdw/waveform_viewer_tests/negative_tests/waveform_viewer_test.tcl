
# Starts the waveform viewer
handlers::start_waveform_viewer

# Attaches the selected waveform viewer
handlers::attach_waveform_viewer Novas

# Loads the dump file
handlers::load_dump_file file

# Reloads the dump file
handlers::reload_dump_file

# Opens the specified project
handlers::open_project project.bspec 

# Specifies Non-Bsv hierarchy for the waveform viewer
#handlers::set_nonbsv_hierarchy /main/top

# Specifies options for waveform viewer
#handlers::set_waveform_viewer Novas -command nWave -options -nologo -close 1

# Attaches the selected waveform viewer
handlers::attach_waveform_viewer

# Starts the waveform viewer
handlers::start_waveform_viewer

# Loads the dump file
handlers::load_dump_file file

# Reloads the dump file
handlers::reload_dump_file

# Attaches the selected waveform viewer
handlers::attach_waveform_viewer wrong

# Starts the waveform viewer
handlers::start_waveform_viewer

# Attaches the selected waveform viewer
handlers::attach_waveform_viewer nWave

# Reloads the dump file
handlers::reload_dump_file

# Loads the dump file
handlers::load_dump_file

# Loads the dump file
handlers::load_dump_file wrong

# Loads the dump file
handlers::load_dump_file ./waveform_viewer_options_test.tcl

# Reloads the dump file
handlers::reload_dump_file

# Closes the current project
handlers::close_project

# Exits the workstation
exit
