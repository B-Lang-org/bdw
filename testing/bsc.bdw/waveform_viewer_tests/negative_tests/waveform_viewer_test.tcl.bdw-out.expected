There is no open project
There is no open project
There is no open project
There is no open project
The viewer is not specified
There is no attached waveform viewer.
There is no viewer running
There is no viewer running
wrong is not supported viewer
There is no attached waveform viewer.
The dump file can not be reloaded
The file for loading is not specified
The wrong file doesn't exist
Unsupported file extension
The dump file can not be reloaded
